engine/runtime-integration-tests/src/test/java/org/enso/compiler/ExecStrictCompilerTest.java

```diff
@@ -106,4 +109,15 @@ public void testUnknownConstructorLocation() throws Exception {
       assertNotEquals("Location defined " + errors, -1, errors.indexOf("wrong_cons:2:5"));
     }
   }
+
+  static void assertContains(String expected, String actual) {
```

Comment:
I can see that you are still fighting hard against adopting hamcrest's simpler `assertThat(text, containsString("...")` or even better pattern for checking multiple substring:
```java
assertThat(someText, allOf(
  containsString("some text"),
  containsString("some other substring")
));
```

Not forcing you to adopt it. Just putting a note that with hamcrest, you don't need to define this `assertContains` method yourself, and you even have better error messages.